### PR TITLE
Make deployment artifiacts with a consistent name pattern.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@leseb @gman0 @Madhu-1 ptal

File: examples/rbd/pod.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: csirbd-demo-pod
```
Comment (ShyamsundarR) [inline examples/rbd/pod.yaml]: deployment.yaml in examples/cephfs/deployment.yaml needs changes to address a similar pattern, IOW changing `csicephfs-demo-depl` to `csi-cephfs-demo-depl`

File: examples/rbd/storageclass.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
-   name: csi-rbd
```
[the remark this answers is not in the document]
Comment (humblec) [reply examples/rbd/storageclass.yaml]: Done.

File: examples/rbd/pod.yaml (hunk 1 above)
Comment (humblec) [reply examples/rbd/pod.yaml]: Done.

**Comment by humblec** [comment]
@Madhu-1 @ShyamsundarR comments are addressed. PTAL .Thanks.

**Comment by Madhu-1** [comment]
@humblec can you send this PR to master branch instead of csiv1.0 branch?

**Comment by ShyamsundarR** [comment]
Changing the PR base branch has brought in more changes than just what was intended in the PR (or I am mistaken). The file list and commit list are now showing more deltas than required.

@humblec please address the same, for now marking this as DNM as I do not want the bot to merge the same.

**Comment by humblec** [comment]
@ShyamsundarR rebased or removed unwanted commit, this should be good now! PTAL

**Pull request merged**